units/systemd-journald.service.in

```diff
@@ -12,7 +12,7 @@ Description=Journal Service
 Documentation=man:systemd-journald.service(8) man:journald.conf(5)
 DefaultDependencies=no
 Requires=systemd-journald.socket
-After=systemd-journald.socket systemd-journald-dev-log.socket systemd-journald-audit.socket syslog.socket
```

Comment:
I think that after https://github.com/systemd/systemd/pull/25687 no changes to units are needed anymore. It should be enough to disable the audit socket.

Reply:
Yep, you are right, that line should be kept as is.